core/src/test/resources/com/linecorp/armeria/server/file/bar/bar.custom-json-extension

```diff
@@ -0,0 +1,3 @@
+{
+    "foo" : "bar"
+}
```

Comment:
nit: Could you make sure all text files end with a newline?

Reply:
when you mention "all text files", Is that mean all 3 files that I added on my commits or all text file under test resources "core/src/test/resources/com/linecorp/armeria/server/file/**" directory. I noticed some of them also don't end with newline as well. 😅

Reply:
All files you added :smile: